### ISSUE TITLE
Activating a bionic temporarily removes stat penalties from wielded artifact.

### ISSUE THREAD

**Issue by GenericPseudonym** [CleverRaven/Cataclysm-DDA#13521]
I have an artifact dagger that gives +20 speed, -3 dex, -3 perception while wielded.

Activating any bionic (including toggling Battery System, which is instant and has no power cost) seems to remove the stat penalties for one turn, although the increased speed stays. My stats go back down to the expected values the turn afterwards.

Active mutations don't have the same effect.

I have the stats through skills mod installed.

OS: Windows 7
Version: 8086029, tiles build
Save file on dropbox: https://www.dropbox.com/sh/xkdyncokl1td3er/AAA1kBC3pdHJQeALMlVSsz_wa

Reproduction: 
1. Activate any active bionic
2. Observe stats in top right corner: Dex changes from 13 to 16, Per changes from 9 to 12
3. Pass one turn.
4. Observe stats changing back to 13 and 9

**Comment by drbig** [comment]
> I have the stats through skills mod installed.

Can you replicate the above without the mod? (just as a sanity check and to rule it out)

**Comment by GenericPseudonym** [comment]
Yes, I just successfully replicated the bug in a new world with no mods.

**Comment by KorGgenT** [comment]
does this still happen?

**Comment by TheGamingFox1911** [comment]
https://cdn.discordapp.com/attachments/598523535169945607/682645968109305912/2020-02-27_11-52-21.mp4
The bug does still happen as of version 0.d-12029-g29de372
(build 10303)

**Comment by Maleclypse** [comment]
Would this still happen now that artifact code has changed to relic code?

**Comment by anoobindisguise** [comment]
I don't seem to see this happening anymore. I can activate my bionics just fine and my backpack full of artifacts doesn't change.
